nifi-nar-bundles/nifi-poi-bundle/nifi-poi-processors/src/test/resources/alternatedelimiters.csv

```diff
@@ -0,0 +1,9 @@
+NumbersTimestampsMoney
+1234.4561/1/17$   123.45
+1234.4612:00:00 PM£   123.45
+1234.5Sunday, January 01, 2017¥   123.45
+1,234.461/1/17 12:00$   1,023.45
+1,234.456012:00 PM£   1,023.45
+9.88E+082017/01/01/ 12:00¥   1,023.45
+9.877E+08
+9.8765E+08
```

Comment:
Without style, the original value is '987654321'. I imagine there might be use-cases in which user would like to get the original value representation. Providing one more processor property to toggle whether apply style table might be more user friendly. How do you think?

Reply:
I looked around online, but the only method I could find to do this would be to create a custom copy of `XSSFSheetXMLHandler` and add in this functionality.